app/models/cache/Cache.scala

```diff
@@ -27,20 +27,6 @@ case class Cache(providerId: String, answers: Option[Answers], queryUcr: Option[
 }
 
 object Cache {
-  implicit private val formatInstant: OFormat[Instant] = new OFormat[Instant] {
```

Comment:
Are you sure this is not required?

I remember that we had a problem with Instant, but not sure what was the problem.

Reply:
All tests passed (ATs too). Might not be an issue anymore.
Removed that after being pointed out by the compiler as unused.